arangod/RocksDBEngine/RocksDBIncrementalSync.cpp

```diff
@@ -342,9 +345,9 @@ Result syncChunkRocksDB(DatabaseInitialSyncer& syncer,
           auto [documentId, revisionId] = lookupResult;
 
           tempBuilder->clear();
-          r = physical->lookupDocument(*trx, documentId, *tempBuilder,
-                                       /*readCache*/ true, /*fillCache*/ false,
-                                       ReadOwnWrites::yes);
+          auto r =
```

Comment:
This assigns the lookup result to a new in-scope variable `r`, whereas the old code assigned the lookup result to a variable `r` from an outer scope.
Is this change intentional?

Reply:
No, it's not, thanks!